### PR TITLE
Fix run build fail after 'npm run generate' 

### PR DESCRIPTION
**Pull request by timtong1982**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Since the types/prop-types updated to 15.5.5, some of the test cases will fail to build due to interface change, resolving is locking the types/prop-types to 15.5.3

Update \tasks\build-codepen-examples.js to address a corner case of when put fabric code in a child folder of 'src', **codepen** task will not correctly generate the **codepen** files to the correct directory.

#### Focus areas to test

(optional)

### CODE REVIEW

File: scripts/package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "github": "^7.0.0",
     "glob": "^7.1.2",
     "gzip-size": "^3.0.0",
-    "jest": "^23.1.0",
+    "jest": "23.0.0",
```
Comment (kenotron) [inline scripts/package.json]: Is this necessary? If so, at least peg it to the version we're using currently which is 23.5.0 it looks like from your shrinkwrap change

File: scripts/tasks/build-codepen-examples.js [hunk 2]
```diff
@@ -24,18 +24,20 @@ module.exports = function() {
...
           const api = { jscodeshift: jscodeshift.withParser('babylon'), stats: {} };
           const transformResult = transformer(fileInfo, api);
-          const dirPath = path.dirname(path.dirname(file)).replace(/(\b)src(\b)/, '$1lib/codepen$2');
+          const dirPath = path.dirname(path.dirname(file)).replace(/((\b)src(\b))(?!.*\1)/, '$2lib/codepen$3');
 
           if (!fs.existsSync(dirPath)) {
             mkdirp.sync(dirPath);
           }
 
+          console.log(path.dirname(file));
```
[the remark this answers is not in the document]
Comment (timtong1982) [reply scripts/tasks/build-codepen-examples.js]: oh.. yes

[the remark this answers is not in the document]
Comment (timtong1982) [reply scripts/tasks/build-codepen-examples.js]: removed.

**Comment by timtong1982** [comment]
Thanks man!
